board/piksiv3/rootfs-overlay/etc/init.d/S86skylark1_upload_daemon

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Are they at the same init level? I thought these files are running in some kind of lexographic ordering?